moveit_planners/ompl/ompl_interface/src/ompl_planner_manager.cpp

```diff
@@ -47,17 +47,58 @@
 #include <moveit_msgs/DisplayRobotState.h>
 #include <moveit_msgs/DisplayTrajectory.h>
 
+#include <ompl/util/Console.h>
+
 #include <memory>
 
 namespace ompl_interface
 {
 using namespace moveit_planners_ompl;
 
+#define OMPL_ROS_LOG                                                                                                   \
+  {                                                                                                                    \
+    ROSCONSOLE_DEFINE_LOCATION(true, ::ros::console::levels::Debug, ROSCONSOLE_NAME_PREFIX ".ompl");                   \
```

Comment:
It looks like this should be:
```c++
#define OMPL_ROS_LOG(ros_level) \
 { \
   ROSCONSOLE_DEFINE_LOCATION(true, ros_level, ROSCONSOLE_NAME_PREFIX ".ompl"); \
...
```
Is that correct?

Reply:
Well spotted. I pushed the fix. If you agree, I will do the final merge.